vlib/crypto/ecdsa/README.md

```diff
@@ -2,7 +2,27 @@
 
 `ecdsa` module for V language. Its a wrapper on top of openssl ecdsa functionality.
 Its currently (expanded) to support the following curves:
+
 - NIST P-256 curve, commonly referred as prime256v1 or secp256r1
 - NIST P-384 curve, commonly referred as secp384r1 
 - NIST P-521 curve, commonly referred as secp521r1
-- A famous Bitcoin curve, commonly referred as secp256k1
\ No newline at end of file
+- A famous Bitcoin curve, commonly referred as secp256k1
+
+# Example
+```codeblock
```

Comment:
Instead of `codeblock`, use `v` here.
The benefit is that the tool `v check-md .` will will check the example on the CI,
and that way it will be kept valid, and also consistent with the rest of the V examples in documentation (formatted, vetted, etc).

Reply:
Okey .. would be changed